sdk-springboot/src/main/java/io/dapr/springboot/DaprTopicSubscription.java

```diff
@@ -43,10 +44,10 @@ class DaprTopicSubscription {
    * @param topic The topic to subscribe to.
    * @param route Destination route for messages.
    * @param routes Destination routes with rules for messages.
-   * @param metadata Metdata for extended subscription functionality.
+   * @param metadata Metadata for extended subscription functionality.
    */
-  DaprTopicSubscription(String pubsubName, String topic, String route,
-                               DaprTopicRoutes routes, Map<String, String> metadata) {
+  DaprTopicSubscription(String pubsubName, String topic, String route, DaprTopicRoutes routes,
```

Comment:
Add an overloaded method to keep the previous signature and backwards compatibility.